test/helpers/Utils.ts

```diff
@@ -29,6 +29,17 @@ export const getBalance = async (token: TokenWithAddress, account: string | Addr
     return await (await Contracts.ERC20.attach(tokenAddress)).balanceOf(accountAddress);
 };
 
+export const getEvent = async (tx: ContractTransaction, eventName: string) => {
+    const events = (await tx.wait()).events;
+
+    const filteredEvents = events?.filter((e) => e.event === eventName);
+    if (filteredEvents === undefined) {
+        return [];
+    } else {
+        return filteredEvents;
+    }
+};
```

Comment:
Minor:
```
    const events = (await tx.wait()).events || [];
    return events.filter((e) => e.event === eventName);
```

Reply:
addressed - better in this case is the ?? operator.

Reply:
addressed - better in this case is the ?? operator.